client/pcr.go

```diff
@@ -26,7 +26,7 @@ const (
 // CertifyHashAlgTpm is the hard-coded algorithm used in certify PCRs.
 const CertifyHashAlgTpm = tpm2.AlgSHA256
 
-func min(a, b int) int {
+func minimum(a, b int) int {
```

Comment:
these needed to change to make the Revive linter happy

Reply:
Min is a builtin golang function starting 1.21. Do you need to define this?

Reply:
Oh cool, no we don't. Ill remove it.